src/schema.ts

```diff
@@ -0,0 +1,21 @@
+export const typeDefs = `
```

Comment:
```suggestion
export const typeDefs = `#graphql
```

Esse comentário o Apollo recomenda. Ele dá um highlight de sintaxe nessa sua stringzona, pode ajudar na leitura e construção do seu schema.